daprdocs/content/en/developing-applications/building-blocks/pubsub/subscription-methods.md

```diff
@@ -192,6 +193,118 @@ func eventHandler(ctx context.Context, e *common.TopicEvent) (retry bool, err er
 
 The `/checkout` endpoint matches the `route` defined in the subscriptions and this is where Dapr sends all topic messages to.
 
+### Streaming subscriptions
+
+Streaming subscriptions are subscriptions defined in application code that can be dynamically stopped and started at runtime.
+Messages are pulled by the application from Dapr. This means no endpoint is needed to subscribe to a topic, and it's possible to subscribe without any app configured on the sidecar at all.
+Any number of pubsubs and topics can be subscribed to at once.
+As messages are sent to the given message handler code, there is no concept of routes or bulk subscriptions.
+
```

Comment:
Given the "there is no concept of routes or bulk subscriptions" Can you update this sentence 

"This feature is available to both the [declarative and programmatic subscription approaches], however does not apply to streaming subscriptions. "

in this doc 
https://docs.dapr.io/developing-applications/building-blocks/pubsub/howto-route-messages/

And need to update here https://docs.dapr.io/developing-applications/building-blocks/pubsub/pubsub-bulk/#subscribing-messages-in-bulk to say that this does not apply to streaming subscriptions in the text.

Reply:
@msfussell Please take a look!